modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -10,36 +10,29 @@
     #liferay.workspace.bundle.cache.dir=~/.liferay/bundles
 
     #
+    # Legacy Feature (See liferay.workspace.product.key)
     # Set the URL pointing to the bundle Zip to download. If the URL points to a
     # DXP bundle (e.g., https://api.liferay.com/...), The default value is the
     # URL for Liferay Portal CE 7.0 GA7, Liferay Portal CE 7.1 GA4, Liferay
     # Portal CE 7.2 GA2, or Liferay Portal CE 7.3 GA1, depending on the portal
     # version the workspace is targeting.
     #
-#if (${liferayVersion.startsWith("7.0")})
-    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip
-#elseif (${liferayVersion.startsWith("7.1")})
-    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.1.3-ga4/liferay-ce-portal-tomcat-7.1.3-ga4-20190508171117552.tar.gz
-#elseif (${liferayVersion.startsWith("7.2")})
-    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.2.1-ga2/liferay-ce-portal-tomcat-7.2.1-ga2-20191111141448326.tar.gz
-#elseif (${liferayVersion.startsWith("7.3")})
-    liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.3.2-ga3/liferay-ce-portal-tomcat-7.3.2-ga3-20200519164024819.tar.gz
-#end
+    #liferay.workspace.bundle.url=https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip
```

Comment:
lets just show the latest GA, so 7.3.2-ga3.  No need to list all the previous GAs.  they are publicly available here:

https://releases-cdn.liferay.com/portal/